benchopt/plotting/default_plots.py

```diff
@@ -0,0 +1,63 @@
+from .base import BasePlot
+
+
+class ObjectiveCurvePlot(BasePlot):
+    name = "Objective Curve"
+    type = "scatter"
+    dropdown = {
+        "dataset": ...,
+        "objective": ...,
+        "objective_column": ...,
+        "X_axis": ["Time", "Iteration"],
+    }
+
+    def plot(self, df, dataset, objective, objective_column, X_axis):
+        df = df[df['dataset_name'] == dataset]
+        df = df[df['objective_name'] == objective]
+
+        plots = []
+        for solver in df['solver_name'].unique():
+            df_filtered = (
+                df[df['solver_name'] == solver]
+                .select_dtypes(include=['number'])
+                .groupby('stop_val')
+            )
+            df_filtered_median = df_filtered.median()
+            if objective_column not in df_filtered_median:
+                continue
+            y = (
+                df_filtered_median[objective_column]
+                .values.tolist()
+            )
+            x = df_filtered_median["time"].values.tolist()
+            if X_axis == "Iteration":
+                x = df_filtered_median["time"].index.tolist()
```

Comment:
I don't think this is the right value?
```suggestion
                x = df_filtered_median["stop_val"].index.tolist()
```